### ISSUE TITLE
KeyError: 'artist'

### ISSUE THREAD

**Issue by chebro** [GuardKenzie/miniplayer#4]
**Description:**

I recently setup mpd with ncmpcpp client, and I found your repo yesterday, but there seems to be a problem here, I've tried playing a song with no album art and I'm faced with a KeyError:
```
miniplayer
Traceback (most recent call last):
  File "/sbin/miniplayer", line 502, in loop
    self.handleKeypress()
  File "/sbin/miniplayer", line 271, in handleKeypress
    if self.checkSongUpdate() == 1:
  File "/sbin/miniplayer", line 241, in checkSongUpdate
    self.artist = song["artist"]
KeyError: 'artist'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/sbin/miniplayer", line 536, in <module>
    player.loop()
  File "/usr/lib/python3.9/site-packages/ueberzug/lib/v0/__init__.py", line 378, in decorator
    return function(*args, canvas=self, **kwargs)
  File "/sbin/miniplayer", line 530, in loop
    if error:
UnboundLocalError: local variable 'error' referenced before assignment
```

I'm using the AUR package. I've tried switching the `image_method` (obviously didn't help because I have no album art).

**Comment by GuardKenzie** [comment]
Fixed in 0491341.

**Comment by GuardKenzie** [comment]
New version is now live on PyPi
